config/dcp.go

```diff
@@ -277,6 +286,16 @@ func (c *Dcp) applyDefaultGroupMembership() {
 	if c.Dcp.Group.Membership.Type == "" {
 		c.Dcp.Group.Membership.Type = MembershipTypeCouchbase
 	}
+
+	if c.Dcp.Group.Membership.Type == MembershipTypeCouchbase {
```

Comment:
i forget to say it in session but we can add unit test for this condition to here -> https://github.com/Trendyol/go-dcp-client/blob/b943b9f5a4fc89c7e74667c970724a433ae65256/config/dcp_test.go#LL146C3-L146C3